features/languages.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Languages
+@QA @Languages
+Scenario: Ensure you can change languages in CommCare
+	Then I install the ccz app at "languages.ccz"
+	Then I login with username "clark" and password "123"

```

Comment:
Can you replace `clark` with `user_with_no_data` so we for sure don't pull any case data during the restore?